backend/test/logger.test.ts

```diff
@@ -13,8 +13,14 @@ const commonConfig = Object.freeze({
 });
 
 describe('logger', () => {
+    function mockConsoleLog() {
+        // @ts-ignore Neither using jest.spyOn(console, 'log') nor global.console = { ...global.console, log: jest.fn() }
+        // works with Winston logger
+        global.console = { log: jest.fn() };
```

Comment:
The only way to avoid using `ts-ignore` I found is to use similar solution as for other mock castings:
```suggestion
        global.console = <typeof console>(<unknown>{ log: jest.fn() });
```